src/libs/actions/BankAccounts.js

```diff
@@ -173,6 +173,7 @@ function validateBankAccount(bankAccountID, validateCode) {
             key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
             value: {
                 isLoading: true,
+                error: '',
```

Comment:
Why do we need to add this? Your changes to Form are handling the `errors` key, no?